share/spice/timer/timer.css

```diff
@@ -1,233 +1,321 @@
-.zci--timer .timer__btn {
-    height: 75px;
-    width: 75px;
-    border-radius: 37.5px;
-    text-align: center;
-    line-height: 75px;
-    border: 1px solid black;
-    font-size: 13px;
-    font-weight: bold;
-    margin-left: 5px;
-    margin-right: 5px;
-    outline: none; /*remove the blue outline in chrome/opera*/
-    background: none;
-    cursor: pointer;
+/* GENERAL */
+
+.zci--timer .cw {
+    max-width: 90em;
 }
 
-/*remove the dashed line in FF/IE*/
-.zci--timer .timer__btn::-moz-focus-inner {
-    border: 0;
+.zci--timer .zci__main {
+    max-width: 100%;
 }
-    
-.zci--timer .timer__btn.timer__start {
-    border-color: #20b84c;
-    background-color: #20b84c;
-    color: #fff;
+
+.zci--timer .zci__body {
+    padding-left: 0;
 }
 
-.zci--timer .timer__btn.timer__pause {
-    border-color: #ff3908;
-    background-color: #ff3908;
-    color: #fff;
+.zci--timer .timer_container {
+    position: relative;
+    width: 305px;
+    height: 140px;
+    display: block;
 }
 
-.zci--timer .timer__btn[disabled] {
-    color: #b4b4b4;
-    border-color: #b4b4b4;
-    cursor: default;
+.zci--timer .play_pause,
+.zci--timer .timer_container,
+.zci--timer #add_timer_container,
+.zci--timer .display_container {
+    float: left;
 }
 
-.zci--timer .timer__btn.timer__start[disabled] {
-    background-color: #ccc;
-    border-color: #ccc;
-    color: #f2f2f2;
+.zci--timer #add_timer_container {
+    display: block;
+    width: 140px;
+    height: 140px;
+    background-color: #e6e6e6;
+    border: none;
+}
+
+.zci--timer a#add_timer_btn {
+    width: 70%;
+    height: 70%;
+    border-radius: 50%;
+    border: 4px solid #dedede;
+    color: #626262;
+    text-decoration: none;
+    display: block;
+    margin: 10% 0 0 10%;
+    text-align: center;
+    font-size: 4em;
+    line-height: 2em;
+    cursor: pointer;
 }
 
-.zci--timer .spice-pane {
+/* PLAY/PAUSE BTN */
+
+.zci--timer .play_pause {
     width: 45%;
     height: 100%;
-    line-height: 65px;
-    display: none; /* we'll display the divs in javascript */
-    padding: 2%;
-    border-radius: 2px;
-    text-align: center;
-    font-family: sans-serif;
+    position: relative;
 }
 
-.zci--timer .spice-pane-right {
-    float: right;
+.zci--timer .play_pause .progress {
+    position: relative;
+    left: 1.5em;
+    top: 1.5em;
+    width: 100px;
+    height: 100px;
+    border: 4px solid #eee;
+    box-sizing: border-box;
+    border-radius: 50%;
 }
 
-.zci--timer .pane-block {
+.zci--timer .timer_container.status_running .play_pause .progress,
+.zci--timer .timer_container.status_paused .play_pause .progress {
+    border-color: #eee;
+}
+
+.zci--timer .timer_container.status_stopped .play_pause .progress {
+    border-color: #de5a2a;
+}
+
+.zci--timer .play_pause .fill {
+    display: none;
+    top: -4px;
+    left: -4px;
+    position: absolute;
+    width: 100px;
     height: 100px;
-    margin-left: 5px;
-    margin-right: 5px;
+    clip: rect(0, 100px, 100px, 50px);
```

Comment:
I haven't seen this property before! MDN seems to suggest we should use `clip-path` instead, but that seems to be a largely experimental/unsupported feature.

https://developer.mozilla.org/en-US/docs/Web/CSS/clip